README.md

```diff
@@ -1 +1,4 @@
-# caraml-sdk
\ No newline at end of file
+# CaraML SDK
+
+Python SDK for interacting with the Merlin and Turing SDKs of CaraML.
```

Comment:
Nit: The `caraml-sdk` is not yet built, we're only building the collecting of common dependencies under it now. So, shall we rephrase as:

> Common Python SDK dependencies for CaraML components